src/Filters/OrdersByCourseRunFilter.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Balfour\LaravelQueryFilters\Filters;
+
+class OrdersByCourseRunFilter extends SingleFieldFilter
+{
+    /**
+     * @param mixed $query
+     * @param mixed $value
+     */
+    public function apply($query, $value)
+    {
+        if ($value === 'null') {
+            $query->whereNull($this->field);
+        } else {
+            $query->leftJoin('delegate_information', 'order_items.id', '=', 'delegate_information.order_item_id')
+                ->where('delegate_information.course_run_id', $value);
```

Comment:
This library should be completely DB agnostic. It doesn't know anything about any tables or fields. You can't reference a specific table or field here.